### ISSUE TITLE
Release tag and npm version out of sync

### ISSUE THREAD

**Issue by QYuQianchen** [Uniswap/v3-periphery#336]
## Description
NPM package is release at version v1.4.3 https://www.npmjs.com/package/@uniswap/v3-periphery, while the release tag on Github is only v1.3.0. 

Keeping them in sync ensures that the same code gets imported in hardhat (via npm) and in foundry (via Github)

## Possible solution
Add a github action that releases to npm automatically on tag push (props to @delaaxe for the idea)

**Comment by drLis** [comment]
Agree. Please update tags for solidity 0.8 branchs, because all we need it for correct working of brownie pm manager (can works only with tag)

**Comment by startswithaj** [comment]
Is there a plan to release 1.4.2-solc-0.8 version to NPM?

**Comment by startswithaj** [comment]
v3-core has a solc 8 version released:
<img width="809" alt="Screenshot 2023-07-02 at 12 39 05 pm" src="https://github.com/Uniswap/v3-periphery/assets/3658522/2e44839d-3c02-4296-8c0b-a23b8adcb961">
